Hazel.UnitTests/StressTests.cs

```diff
@@ -26,5 +30,44 @@ public void StressTestOpeningConnections()
                 connection.Connect(new byte[5]);
             });
         }
+
+        // This was a thing that happened to us a DDoS. Mildly instructional that we straight up ignore it.
+        public void SourceAmpAttack()
```

Comment:
Noting no TestMethod attribute--just here to test optionally?

Reply:
Yeah, this class doesn't really contain tests so much as playtoys.